libs/platform/i18n/src/lib/locale/default-locale.service.spec.ts

```diff
@@ -58,25 +58,6 @@ describe('DefaultLocaleService', () => {
 
       expect(cb).toHaveBeenCalledWith('mock-locale');
     });
-
-    it('should not emit LocaleChanged query with locale initially', () => {
```

Comment:
I'd expect some more unit tests after a bug fix, not only removing existing ones ;)

Reply:
But it's already covered in `set` section.